src/main/java/interface_adaptors/sales_report/SalesReporterController.java

```diff
@@ -0,0 +1,33 @@
+package interface_adaptors.sales_report;
+
+import use_cases.gateway_interfaces.gatewayReaderInterface;
+import use_cases.sales_report.SalesReporterInputBoundary;
+import use_cases.sales_report.SalesReporterInputData;
+
+import java.io.IOException;
+import java.util.List;
+
+public class SalesReporterController {
+    private final SalesReporterInputBoundary salesReportUseCase;
+    private final GatewayReaderInterface reader;
+    private final String FILE_PATH =
+            "C:\\Users\\leste\\OneDrive - University of Toronto\\School\\Fall 2022\\CSC 207\\course-project-group-131\\src\\main\\java\\database\\Sample Data - Sample History.csv";
```

Comment:
Maybe change this line to Path from Repository Root, (src\\main\\java\\database\\Sample Data - Sample History.csv)